data/sql/updates/pending_db_world/rev_1643745379505594100.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643745379505594100');
+DELETE FROM `creature` WHERE  `guid` = 247680; # Duplicate of 247673
```

Comment:
instead of using # can you please put them in a commit out such as inbetween /*      */, 
for example ```/* Duplicate of 247673 */```